sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalogSuite.scala

```diff
@@ -1119,4 +1119,40 @@ class V2SessionCatalogNamespaceSuite extends V2SessionCatalogBaseSuite {
     }
     catalog.dropNamespace(testNs, cascade = false)
   }
+
+  test("IdentifierHelper should throw exception when identifier has more than 2 parts") {
+    val testIdent: IdentifierHelper = Identifier.of(Array("a", "b"), "c")
```

Comment:
Can we have a test with examples of queries that users might actually use, such as SQL tests?

It would be great if we have a test in `sql/core/src/test/scala/org/apache/spark/sql/errors/QueryCompilationErrorsSuite.scala` if possible.

Reply:
@itholic done, please check.
Are these tests redundant? Or do we need to keep them?

Reply:
I think it's okay to just leave it as it is.